src/main/java/com/dkohut/chatservice/common/entity/ChatRoom.java

```diff
@@ -0,0 +1,61 @@
+package com.dkohut.chatservice.common.entity;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+
+/**
+ * 
+ * @author Dmytro Kohut
+ *
+ */
+public class ChatRoom {
+	
+	private Integer id;
+	private String name;
+	private Timestamp createdAt;
+	
+	/**
+	 * This constructor will be used when receiving an object from HTTP-request. 
+	 */
+	public ChatRoom() {
+		// default constructor
+	}
+	
+	/**
+	 * This constructor will be used in DAO methods.
+	 * 
+	 * @param ResultSet resultSet
+	 * @throws SQLException 
+	 */
+	public ChatRoom(ResultSet resultSet) throws SQLException {
```

Comment:
I would not recommend throw an exception in constructor, better is factory method.

Reply:
It's done.